sdk/eventhub/eventhubs-checkpointstore-table/test/tables-checkpointstore.spec.ts

```diff
@@ -0,0 +1,518 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import chai from "chai";
+import * as dotenv from "dotenv";
+const should = chai.should();
+import { TableCheckpointStore } from "../src";
+import debugModule from "debug";
+const debug = debugModule("azure:event-hubs:partitionPump");
```

Comment:
This looks like copy-paste code that can be removed.

Reply:
i actually used debug in my tests , similar to how it was used in tests for blobs.

Reply:
Ah, I think the string you're passing in should be updated, since this isn't the event-hubs partition pump :smile: